tests/test_task_agnostic.py

```diff
@@ -0,0 +1,12 @@
+"""
+Test functions for the task_agonstic.py metrics.
+"""
+
+from locomoset.metrics.task_agnostic import num_params_metric
+
+
+def test_num_params_metric(dummy_model_classifier):
```

Comment:
I'd be more inclined to use `dummy_model_head` here (or have a test with both showing that you get a different no. of parameters in each case), since we decided using the model head is probably more generalisable/predictive for what we're interested in.

Reply:
I ran some tests on this and they it does get different numbers of parameters, but not in a significant way across the models. I've now changed it to `dummy_model_head` and updated the test.